### ISSUE TITLE
Add SPM Support

### ISSUE THREAD

**Issue by Ganesh623** [ApplikeySolutions/VegaScroll#20]

**Comment by rawnly** [comment]
Yup it would be awesome! It's also [super easy](https://medium.com/@francisco.gindre/supporting-swift-package-manager-in-your-pod-in-5-simple-steps-d4b76ea80470) to do!

**Comment by beelabs-co** [comment]
Is there any update for SPM?

**Comment by rawnly** [comment]
@beelabs-co I created a PR (#22) for that, but @AppliKey isn't merging it 😞
